modules/economy-system/cli.js

```diff
@@ -44,7 +44,7 @@ module.exports.commands = [
         command: 'balance',
         description: 'Show all balances from the DataBase',
         run: async function (input) {
-            const balances = await input.client.models['economy-system']['Balance'].findAll();
+            const balances = await input.client.models['economy-system']['NewBalance'].findAll();
```

Comment:
I would suggest naming the old model `BalanceOld` (or similar) and leave the name of the current model at `Balance`. 

(this is just a suggestion, you can leave it as it is, if you want)